src/styles/review.css

```
@@ -0,0 +1,54 @@
+
+.card-price {
+  display: flex;
+  justify-content: space-between;
+  align-items: center;
+  margin: 0 16px 16px;
+}
+
+.card-price__text {
+  font-weight: 400;
+  font-size: 12px;
+  color: #616070;
+  margin: 0;
+}
+
+.card-price__value {
+  font-weight: 700;
+  margin: 0;
+}
```

```suggestion
.card-price {
  display: flex;
  justify-content: space-between;
  align-items: center;
  margin: 0 16px 16px;
}

.card-price__text {
  font-weight: 400;
  font-size: 12px;
  color: #616070;
  margin: 0;
}

.card-price__value {
  font-weight: 700;
  margin: 0;
}
```

If it's about styling the card element, than this lines of code should be present in the card.css file and not here. Otherwise, someone else would have difficulties in finding all the code for the card.